### PR TITLE
Update spaceapi crate to 0.7

### PR DESCRIPTION
**Pull request by rnestler**
## Checklist

- [x] Tests added if applicable
- [x] README updated if applicable
- [x] CHANGELOG updated if applicable
- [x] If a commit includes a breaking change, include the string `[breaking-change]` somewhere in the commit message

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,7 +14,8 @@ Possible log types:
 ### Unreleased
 
 - [changed] Bump dependencies, including spaceapi which now uses an enum for
-  the `issue_report_channels` (#92)
+  the `issue_report_channels` and changes the in momory format of extensions
```
Comment (dbrgn) [inline CHANGELOG.md]:
```suggestion
  the `issue_report_channels` and changes the in-memory format of extensions
```

Is the in-memory format relevant at all for users of this crate? I don't think so.

**Comment by g5pw** [comment]
Hey guys! Do you think you can postpone releasing this until the last day of camp? I think I could have the error_chain refactor ready by then!

**Comment by dbrgn** [comment]
@g5pw sure! note that this PR only updates the spaceapi dependency, it does not yet release spaceapi-server itself yet.

**Comment by g5pw** [comment]
Oh, sorry @dbrgn I misread :)

File: CHANGELOG.md (hunk 1 above)
Comment (rnestler) [reply CHANGELOG.md]: I rewrote the entry completely.

**Pull request merged**